bobtemplates/odoo/security/security/rules_+security.model.name_underscored+.xml.bob

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright {{{ copyright.year }}} {{{ copyright.name }}}
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+{{% if odoo.version < 9 %}}
+<openerp>
+<data>
+{{% else %}}
+<odoo>
+{{% endif %}}
+
+    <record model="ir.rule" id="{{{ security.model.name_underscored }}}_name_rule"> <!-- TODO rule id -->
```

Comment:
Include this in a single file with ACL

Reply:
I would say there are two schools of thought:

* ACL's and record rules in the same file (`{model_name}.xml`)
* ACL's in a file (`acl_{model_name}.xml`) and record rules in another one (`rules_{model_name}.xml`)

Maybe we can add a question "ACL and rules in the same file?" that would adapt the behavior accordingly. wyt?

Reply:
To satisfy everyone. I will change my MR to add question  "ACL and rules in the same file?".